web-app/js/portal/cart/GogoduckV1DownloadHandler.js

```diff
@@ -2,8 +2,34 @@ Ext.namespace('Portal.cart');
 
 Portal.cart.GogoduckV1DownloadHandler = Ext.extend(Portal.cart.AsyncDownloadHandler, {
 
-    _getDownloadOptionTextKey: function() {
-        return 'downloadAsSubsettedNetCdfLabel';
+    getDownloadOptions: function() {
+
+        var downloadOptions = [];
+
+        if (this._hasRequiredInfo()) {
+
+            downloadOptions.push({
+                textKey: this._getDownloadAsCdfKey(),
+                handler: this._getUrlGeneratorFunction(),
+                handlerParams: {
+                    asyncDownload: true,
+                    collectEmailAddress: true,
+                    serviceResponseHandler: this.serviceResponseHandler
+                }
+            });
+
+            downloadOptions.push({
+                textKey: this._getDownloadAsCsvKey(),
+                handler: this._getUrlGeneratorFunction(),
+                handlerParams: {
+                    asyncDownload: true,
+                    collectEmailAddress: true,
+                    serviceResponseHandler: this.serviceResponseHandler,
+                    downloadFormat: 'CSV'
```

Comment:
I'm not intending to review this PR - but just so you know GGD V1 will not support CSV download.